### PR TITLE
[flant-integration] Describe what info Deckhouse sends

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Describes what information Deckhouse sends.

## Changelog entries
```changes
section: flant-integration
type: feature
summary: Described what information Deckhouse sends.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/600-flant-integration/docs/README_RU.md [hunk 1]
```diff
@@ -103,3 +93,99 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
...
+Графики по метрикам из upmeter в Grafana:
+![](../../images/600-flant-integration/image3.png)
+
+В среднем модуль upmeter отправляет **70 строк каждые 30 секунд**.
+
+### Как отключить отправку данных Deckhouse? 
+
+Чтобы отключить отправку данных, необходимо:
+- Отключить модуль `flantIntergation`, чтобы отключить регистрацию в Madison и отправку метрик.
+- Удалить CustomResource [UpmeterRemoteWrite](../500-upmeter/cr.html#upmeterremotewrite), чтобы отключить отправку [метрик доступности](#метрики-sla) модулем [upmeter](../500-upmeter/).
```
Comment (z9r5) [inline ee/modules/600-flant-integration/docs/README_RU.md:186]: @nabokihms Am I right here?

File: ee/modules/600-flant-integration/docs/README_RU.md [hunk 2]
```diff
@@ -103,3 +93,99 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
...
+В логах **нет секретных данных** касающихся кластера (примеры сообщений приведены на рисунке ниже). Собранная информация помогает понять, какие действия выполнял оператор и когда он их выполнял.
+
+Пример логов оператора Deckhouse:
+![](../../images/600-flant-integration/image4.png)
+
+При смене релиза Deckhouse, количество отправляемых данных логов достигает в среднем **150 записей в минуту**. При обычной работе, количество отправляемых данных логов достигает в среднем **20 записей в минуту**.
+
+#### Метрики SLA
+
+С помощью модуля [upmeter](../500-upmeter/) в кластере выполняется измерение уровня доступности различных компонентов. Такие метрики необходимы для определения уровня доступности компонентов платформы и анализа выполнения соглашения об уровне сервиса (SLA, Service Level Agreement).
```
Comment (z9r5) [inline ee/modules/600-flant-integration/docs/README_RU.md:170]: This section refers to the upmeter module.  Maybe it's better to move this block to the upmeter module and link it here?
@nabokihms

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/600-flant-integration/docs/README_RU.md [hunk 3]
```diff
@@ -4,47 +4,37 @@ title: "Модуль flant-integration"
 
 ## Описание
 
-Модуль выполняет функции по интеграции с различными сервисами Flant:
+Модуль выполняет функции по интеграции с различными сервисами Флант:
+* Устанавливает в кластер madison-proxy в качестве alertmanager для Prometheus. Регистрируется в [Madison](#оповещения-в-madison).
+* [Отправляет детальную статистику](#статистика-о-состоянии-кластера), необходимую для расчета стоимости обслуживания кластера.
+* [Отправляет](#логи-оператора-deckhouse) логи оператора Deckhouse, необходимые для облегчения процесса отладки.
```
Comment (shvgn) [inline ee/modules/600-flant-integration/docs/README_RU.md:10]: Я думаю, важно сказать, что лишнего не утекает

```suggestion
* [Отправляет](#логи-оператора-deckhouse) логи оператора Deckhouse, необходимые для облегчения процесса отладки. В логах содержится информация только о компонентах декхауса.
```

File: ee/modules/600-flant-integration/docs/README_RU.md [hunk 4]
```diff
@@ -103,3 +93,98 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
...
+
+Данные, отправляемые из каждого кластера:
+- Статистика о состоянии кластера: 
+  - версия Kubernetes;
+  - версия Deckhouse;
+  - канал обновлений; 
+  - количество узлов, и т.п.
+- Оповещения, отправляемые в систему работы с инцидентами Madison.
+- Метрики SLA по всем компонентам Deckhouse.
+- Логи оператора Deckhouse для отладки.
```
Comment (shvgn) [inline ee/modules/600-flant-integration/docs/README_RU.md:122]: No need to repeat the purpose

```suggestion
- Логи оператора Deckhouse.
```

File: ee/modules/600-flant-integration/docs/README_RU.md [hunk 5]
```diff
@@ -103,3 +93,98 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
...
+- Способ подключения к master-узлам кластера.
+
+#### Статистика о состоянии кластера
+
+При помощи [shell-operator](https://github.com/flant/shell-operator) модуль flant-integration выполняет сбор метрик о состоянии объектов кластера. Затем с помощью [Grafana agent](https://github.com/grafana/agent) собранные метрики передаются по протоколу [Prometheus Remote Write](https://docs.sysdig.com/en/docs/installation/prometheus-remote-write/).
+
+На основании собранных данных:
+- Рассчитывается стоимость услуги [Managed Kubernetes](https://flant.ru/services/managed-kubernetes-as-a-service).
+- Происходит планирование миграций.
```
Comment (shvgn) [inline ee/modules/600-flant-integration/docs/README_RU.md:131]: > Происходит планирование миграций.

This doesn't make any sense

> Контролируется гомогенность инфраструктуры.

And this one, tool.

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 2 above)
Comment (shvgn) [reply ee/modules/600-flant-integration/docs/README_RU.md:169]: I agree. I think it deserves at most a couple of sentences.

> Upmeter sends availablility metrics, that lets Flant control SLA per cluster. The metrics throughput is about **100 samples sent every 5 minutes**, the number of samples being dependent on the set of enabled modules in Deckhouse.

After that we can give a link to dedicated Upmeter page.

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 1 above)
Comment (shvgn) [reply ee/modules/600-flant-integration/docs/README_RU.md:185]: It's enough disable `flant-integration` module. 

When user disables `flant-integration` module, its object are to be deleted by deckhouse including `UpmeterRemoteWrite`. With that, Upmeter stops exporting metrics.  After deckhouse converges all modules, there will be no module-related `UpmeterRemoteWrite` object to delete.

**Review by shvgn — CHANGES_REQUESTED** [review]

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 1 above)
Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Thanks. Refactored.

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 2 above)
Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Thanks. Refactored.

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 3 above)
Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Мне кажется ту это уже лишнее. Добавил пояснения в блок, куда ведет ссылка про отправку логов. Выделил.  Достаточно?

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 4 above)
Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Fixed.

File: ee/modules/600-flant-integration/docs/README_RU.md (hunk 5 above)
Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Fixed

Comment (z9r5) [reply ee/modules/600-flant-integration/docs/README_RU.md]: Fixed

**Comment by z9r5** [comment]
Refactored a bit. Also, the upmeter documentation was updated.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Review by shvgn — APPROVED** [review]
LGTM

File: modules/500-upmeter/docs/README_RU.md [hunk 6]
```diff
@@ -2,11 +2,24 @@
...
+- **agent** — программа, которая периодически запускает пробы и отправляет результаты в агрегатор. Работает на мастер-узлах.
+- **upmeter** — агрегатор результатов и API-сервер для их извлечения. Умеет соединять историю результатов проб с CustomResource `Downtime`, где вручную описываются инциденты.
 - **front**
-    - **status** — показывает текущий уровень доступности за последние 10 минут (по умолчанию требует авторизации, но её можно отключить).
-    - **web-ui** — показывает уровни доступности по пробам во времени после (требует авторизации).
+  - **status** — показывает текущий уровень доступности за последние 10 минут (по умолчанию требует авторизации, но её можно отключить).
+  - **web-ui** — показывает уровни доступности по пробам во времени после (требует авторизации).
 - **smoke-mini** — постоянное *smoke-тестирование* с помощью StatefulSet, похожего на настоящее приложение.
+
+Модуль отправляет примерно 100 sample'ов метрик каждые 5 минут, но это значение зависит от количества включенных модулей Deckhouse.
```
Comment (shvgn) [inline modules/500-upmeter/docs/README_RU.md:17]:
```suggestion
Модуль отправляет около 100 показаний метрик каждые 5 минут. Это значение зависит от количества включенных модулей Deckhouse. 
```

**Review by shvgn — APPROVED** [review]
Upmeter suggestions

File: modules/500-upmeter/docs/README_RU.md [hunk 7]
```diff
@@ -2,11 +2,24 @@
 title: "Модуль upmeter"
 ---
 
-Модуль предназначен для мониторинга состояния кластера и отображения страницы статуса с «уровнем доступности» (SLA).
+Модуль предназначен для определения уровня доступности компонентов кластера и компонентов Deckhouse. Позволяет оценивать степень выполнения условий соглашения об уровне сервиса (SLA, Service Level Agreement) на компоненты кластера и компоненты Deckhouse. Предоставляет web-интерфейс отображающий диаграммы доступности компонентов, с возможностью фильтрации и указания временных интервалов.
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/500-upmeter/docs/README_RU.md]: FIxed

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by z9r5** [comment]
Cherry-picked into release-1.32

**Pull request merged**